frontend/packages/core/src/text.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import { Fab, Grid } from "@material-ui/core";
+
+import { ClipboardButton } from "./button";
+
+const CopyButtonContainer = styled(Grid)({
+  marginLeft: "7px",
+});
+
+const Pre = styled.pre({
+  border: "1px solid rgba(13, 16, 48, 0.38)",
+  backgroundColor: "rgba(13,16,48,0.12)",
+  borderRadius: "4px",
+  fontSize: "16px",
+  color: "#242b8c",
+  padding: "12px 16px",
+  flex: 1,
+  whiteSpace: "pre-wrap",
+  wordWrap: "break-word",
+  flexDirection: "row-reverse",
+  display: "flex",
+  overflowY: "scroll",
+});
+
+interface CodeProps {
+  children: string;
+  showCopyButton?: boolean;
+}
+
+const Code = ({ children, showCopyButton = true }: CodeProps) => (
+  <Pre>
+    {showCopyButton && (
+      <CopyButtonContainer container justify="flex-end">
```

Comment:
ideally the button wouldn't scroll with the view...

Reply:
Not sure this is possible without absolute positioning but since this is within a modal that is much more difficult.

Reply:
i see what you're saying, yeah that's tricky i guess. throw in a TODO for now no need to spend a bunch of time on it